sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/dag/TaskGroup.java

```diff
@@ -337,6 +339,7 @@ public void addPostRunDependentTaskGroup(TaskGroup dependentTaskGroup) {
     /**
      * @return list with current task entries in this task group
      */
+    @SuppressFBWarnings(value = "BC_UNCONFIRMED_CAST_OF_RETURN_VALUE", justification = "Incorrect spot bugs")
```

Comment:
Currently the norm in sdk is to add it to suppression
```
  <Match>
    <Class name="~com\.microsoft\.azure\.keyvault\.requests\.\w+Request"/>
    <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
  </Match>
```

We could keep the annotation, and see what to do after more services get checked.

Reply:
Agree, if much more services meet such an error spotbugs, we could add it to suppression.